tests/browser_test.go

```diff
@@ -50,6 +52,26 @@ func TestBrowserNewPage(t *testing.T) {
 	assert.Equal(t, 0, l, "expected there to be 0 browser context after second page close, but found %d", l)
 }
 
+func TestTmpDirCleanup(t *testing.T) {
+	tmpDirPath := "./"
+
+	err := os.Setenv("TMPDIR", tmpDirPath)
+	assert.NoError(t, err)
+	defer func() {
+		err = os.Unsetenv("TMPDIR")
+		assert.NoError(t, err)
+	}()
+
+	b := newTestBrowser(t, withSkipClose())
+	p := b.NewPage(nil)
+	p.Close(nil)
+	b.Close()
+
+	matches, err := filepath.Glob(tmpDirPath + "xk6-browser-data-*")
+	assert.NoError(t, err)
+	assert.Empty(t, matches, "a file/dir exists still which matches the pattern `xk6-browser-data-*`")
```

Comment:
Can you also do a check before calling `b.Close()` that the temp dir does in fact exist?